pages/linux/efibootbgr.md

```diff
@@ -0,0 +1,23 @@
+# efibootmgr
+
+> Manipulate the UEFI Boot Manager (the Bootoptions).
+
+- List the current settings:
+
+`efibootmgr -v for Filepaths`
```

Comment:
Please use just the simple 'efibootmgr' as the first example. Make `-v` to be the last example.